filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -306,19 +306,23 @@ The default setting is 10s.
 [[scan-sort]]
 ===== scan.sort
 
-Specifies if files should be harvested in order and how to determine the order. Possible values are modtime, filename and none. To sort by file modification time use modtime otherwise use filename.
+[]experimental
 
-If you specify a value other than none for this setting you can determine whether to use ascending or descending order using `scan.order`.
+ If you specify a value other than none for this setting you can determine whether to use ascending or descending order using `scan.order`. Possible values are `modtime` and `filename`. To sort by file modification time, use `modtime`, otherwise use `filename`. Leave this option empty to disable it.
```

Comment:
"other than none" is confusing here, because it used to be the string "none" and now it's the empty string, right? I suggest "other than the empty string".